### PR TITLE
Add Project Loved: December 2021 Newspost

### PR DESCRIPTION
**Pull request by Pachiru**
Need approval before pushing the newspost with forum posts.
Also need to add this banner at the end of the intro:
https://i.imgur.com/dkCTYf1.png

### CODE REVIEW

File: news/2021-12-16-project-loved-december-2021.md [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1130748.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[ExileLord - Obsidian](https://osu.ppy.sh/beatmapsets/1130748#fruits) by [Sutavv](https://osu.ppy.sh/users/5320583)\
+170 BPM, 3:17 | 10.69★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+Sutavv making their first appearance in the Loved category and it is quite an entrance if I have to say so myself.
+
+- Star rating over 10: check\
```
Comment (ZeroPyrozen) [inline news/2021-12-16-project-loved-december-2021.md:251]: Typo?

File: news/2021-12-16-project-loved-december-2021.md [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+Definitely a long overdue gem, a really good one some might even add, so give a round of applause to Ska, and preorder *Bayonetta III* while you're at it, will ya?
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1596213.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Ariabl'eyeS - Ariabl'eyeS Collection](https://osu.ppy.sh/beatmapsets/1596213#mania) by [\[Crz\]Rachel](https://osu.ppy.sh/users/8369409)\
+186 – 319 BPM, 5:48 | 4.64★ – 7.34★\
+The \[Delete upon download! (marathon)\] difficulty is *not* being nominated for Loved.\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+Do you want even more stamina gaming? This wave definitely delivers some long-awaited handstream classics, as we decided to bundle up a few of the famous \\[Crz\\]Rachel Ariabl'eyeS charts and nominate them!
```
Comment (ZeroPyrozen) [inline news/2021-12-16-project-loved-december-2021.md:343]:
```suggestion
Do you want even more stamina gaming? This wave definitely delivers some long-awaited handstream classics, as we decided to bundle up a few of the famous \[Crz\]Rachel Ariabl'eyeS charts and nominate them!
```

File: news/2021-12-16-project-loved-december-2021.md (hunk 1 above)
Comment (Walavouchey) [reply news/2021-12-16-project-loved-december-2021.md:251]: the news post generator must've confused list items for a hatnote

Comment (Walavouchey) [reply news/2021-12-16-project-loved-december-2021.md:251]:
```suggestion
- Star rating over 10: check
```

File: news/2021-12-16-project-loved-december-2021.md [hunk 3]
```diff
@@ -0,0 +1,523 @@
+---
+layout: post
+title: "Project Loved: December 2021"
+date: 2021-12-16 12:00:00 +0000
```
Comment (Walavouchey) [inline news/2021-12-16-project-loved-december-2021.md:4]: yeah that date sure passed

Comment (Pachiru) [reply news/2021-12-16-project-loved-december-2021.md:4]: I'd prefer if Eph takes care of this cause last time I did that, it fucked everything up, lol...

File: news/2021-12-16-project-loved-december-2021.md [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+
+[Tatsh - Xevel -Platinum Long Ver.-](https://osu.ppy.sh/beatmapsets/1160712#fruits) by [Del](https://osu.ppy.sh/users/2996241)\
+176 BPM, 4:19 | 7.45★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+Del is one of those players that mapped a lot but never really tried hard enough to get their maps ranked, leaving a lot of hidden gems behind that remain untouched. *Xevel* is one of those.
+
+The difficulty of this map is rather high because of the unusual style of the map. Even though it utilizes patterns you would also see in one or the other ranked map, they are placed in a way that makes catching them difficult. Doing good on this map requires consistency, good reading and good catcher control. Certainly a map for people that are on another level.
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1191248.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
```
Comment (Walavouchey) [inline news/2021-12-16-project-loved-december-2021.md:235]: This image hasn't been uploaded

File: news/2021-12-16-project-loved-december-2021.md [hunk 5]
```diff
@@ -0,0 +1,525 @@
...
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1130748.jpg)](https://osu.ppy.sh/community/forums/topics/1484644)
+
+[ExileLord - Obsidian](https://osu.ppy.sh/beatmapsets/1130748#fruits) by [Sutavv](https://osu.ppy.sh/users/5320583)\
+170 BPM, 3:17 | 10.69★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+Sutavv making their first appearance in the Loved category and it is quite an entrance if I have to say so myself.
+
+- Star rating over 10: check\
```
Comment (Walavouchey) [inline news/2021-12-16-project-loved-december-2021.md:253]: redo the review commit ty

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2021-12-16-project-loved-december-2021.md [hunk 6]
```diff
@@ -0,0 +1,525 @@
...
+Definitely a long overdue gem, a really good one some might even add, so give a round of applause to Ska, and preorder *Bayonetta III* while you're at it, will ya?
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1596213.jpg)](https://osu.ppy.sh/community/forums/topics/1484637)
+
+[Ariabl'eyeS - Ariabl'eyeS Collection](https://osu.ppy.sh/beatmapsets/1596213#mania) by [\[Crz\]Rachel](https://osu.ppy.sh/users/8369409)\
+186 – 319 BPM, 5:48 | 4.64★ – 7.34★\
+The \[Delete upon download! (marathon)\] difficulty is *not* being nominated for Loved.\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+Do you want even more stamina gaming? This wave definitely delivers some long-awaited handstream classics, as we decided to bundle up a few of the famous \\[Crz\\]Rachel Ariabl'eyeS charts and nominate them!
```
Comment (Walavouchey) [inline news/2021-12-16-project-loved-december-2021.md:345]:
```suggestion
Do you want even more stamina gaming? This wave definitely delivers some long-awaited handstream classics, as we decided to bundle up a few of the famous \[Crz\]Rachel Ariabl'eyeS charts and nominate them!
```

File: news/2021-12-16-project-loved-december-2021.md (hunk 5 above)
Comment (Walavouchey) [reply news/2021-12-16-project-loved-december-2021.md:253]:
```suggestion
- Star rating over 10: check
```

**Review by Walavouchey — DISMISSED** [review]

File: news/2021-12-19-project-loved-december-2021.md [hunk 7]
```diff
@@ -0,0 +1,525 @@
...
+
+Cast your vote and find out how far players, and maybe even you yourself, can push the leaderboards!
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/675918.jpg)](https://osu.ppy.sh/community/forums/topics/1484641)
+
+[Various Artists - Jack Dumps](https://osu.ppy.sh/beatmapsets/675918#mania) by [Seiryuu](https://osu.ppy.sh/users/2808028)\
+145 – 170 BPM, 4:06 | 4.83★ – 5.82★\
+*description written by [RandomeLoL](https://osu.ppy.sh/users/7080063)*
+
+Whether you were someone who stumbled upon this without knowing what even a Dan is \[i\](which I commend you for, you're doing great)\[/i\], or someone who discovered Seiryuu's selection of jack maps after seeing *ceeeertain charts inside* in other courses, it is undeniable that both types of players ended up liking the pack to the point it became quite a popular collection!
```
Comment (Walavouchey) [inline news/2021-12-19-project-loved-december-2021.md:284]:
```suggestion
Whether you were someone who stumbled upon this without knowing what even a Dan is *(which I commend you for, you're doing great)*, or someone who discovered Seiryuu's selection of jack maps after seeing *ceeeertain charts inside* in other courses, it is undeniable that both types of players ended up liking the pack to the point it became quite a popular collection!
```

**Review by Walavouchey — APPROVED** [review]
ok now it's good, actually skimmed for errors

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**